src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -307,6 +307,20 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
         @Config.Comment("The default length of a month (in days) when a new world is started. This can be changed in existing worlds via the /timetfc command.")
         @Config.LangKey("config." + MOD_ID + ".general.defaultMonthLength")
         public int defaultMonthLength = 8;
+
+        @Config.Comment("Should animals became old and die?")
+        @Config.LangKey("config." + MOD_ID + ".general.enableAnimalAging")
+        public boolean enableAnimalAging = true;
```

Comment:
What does being old do to animals - just prevent breeding + reproducing, and have a chance of dying every day?

Reply:
Also prevents produce (milk, egg, wool)